test/test_gpu1.cpp

```diff
@@ -950,14 +950,13 @@ TEST_F(NVFuserTest, FusionOuterSplit_CUDA) {
 
   auto options = at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
 
-  at::Tensor output = at::empty({2, 6, 32}, options);
-
   FusionExecutor fe;
   fe.compileFusion(&fusion);
-  fe.runFusion({}, {output});
+  auto outputs = fe.runFusion({});
+  const auto& output = outputs.at(0);
 
   at::Tensor output_ref = at::zeros_like(output, options);
```

Comment:
```suggestion
  at::Tensor output_ref = at::ones_like(output, options);
```

Reply:
nit: we are doing 1+2+3 instead of 0 +1 + 2 + 3.

Reply:
Makes sense. Changed!